corehq/apps/es/tests/test_ensure_indices_reindexed.py

```diff
@@ -0,0 +1,185 @@
+            EnsureIndicesReindexedCommand().is_doc_count_difference_reasonable('case_search')
+        )
+
+    @patch.object(EnsureIndicesReindexedCommand, 'get_doc_count_delta_percent')
+    def test_is_doc_count_difference_reasonable_diff_lt_1_for_rest_indices(self, patched_fn):
+        patched_fn.return_value = 1
+        self.assertFalse(
+            EnsureIndicesReindexedCommand().is_doc_count_difference_reasonable('apps')
+        )
+
+        patched_fn.return_value = 0.98
+        self.assertFalse(
+            EnsureIndicesReindexedCommand().is_doc_count_difference_reasonable('apps')
+        )
+
+    @patch.object(EnsureIndicesReindexedCommand, 'check_indices_are_consistent')
+    def test_command_skips_if_current_es_version_greater(self, patched_fn):
+        with patch_elastic_version(manager, "5"):
+            call_command(
+                'ensure_indices_reindexed', '--target_es_version', 2,
+                '--changelog', 'http://somechangelog.com'
+            )
+        self.assertFalse(patched_fn.called)
+
+    @patch.object(EnsureIndicesReindexedCommand, '_secondary_index_name')
+    @patch.object(EnsureIndicesReindexedCommand, '_primary_index_name')
+    def test_command_raises_if_indices_missing(self, primary_index_patch, secondary_index_patch):
+        primary_index_patch.return_value = self.primary_index_name
+        secondary_index_patch.return_value = self.secondary_index_name
+
+        manager.index_delete(self.secondary_index_name)
+        with patch_elastic_version(manager, "2"):
+            with self.assertRaises(CommandError):
+                call_command(
+                    'ensure_indices_reindexed', '--target_es_version', 2,
+                    '--changelog', 'http://somechangelog.com'
+                )
+        # adding for the cleanup to continue error free
+        self._create_index(self.secondary_index_name)
+
+    @patch.object(EnsureIndicesReindexedCommand, '_secondary_index_name')
+    @patch.object(EnsureIndicesReindexedCommand, '_primary_index_name')
+    def test_command_do_not_raise_if_indices_count_match(self, primary_index_patch, secondary_index_patch):
+        primary_index_patch.return_value = self.primary_index_name
+        secondary_index_patch.return_value = self.secondary_index_name
+
+        index_cname = 'apps'
+        adapters = [primary_adapter, secondary_adapter] = (
+            EnsureIndicesReindexedCommand().get_both_adapters_for_cname(index_cname)
+        )
+        self._populate(5, adapters)
+
+        with patch_elastic_version(manager, "2"):
+            call_command(
+                'ensure_indices_reindexed', '--target_es_version', 2,
+                '--changelog', 'http://somechangelog.com'
+            )
+
+    @patch.object(EnsureIndicesReindexedCommand, '_secondary_index_name')
+    @patch.object(EnsureIndicesReindexedCommand, '_primary_index_name')
+    def test_command_raises_if_indices_count_match(self, primary_index_patch, secondary_index_patch):
```

Comment:
Is this test name correct?
```suggestion
    def test_command_raises_if_indices_count_does_not_match(self, primary_index_patch, secondary_index_patch):
```